src/backend/dx11/src/device.rs

```diff
@@ -933,7 +1001,7 @@ impl hal::Device<Backend> for Device {
             requirements: memory::Requirements {
                 size: size,
                 alignment: 1,
-                type_mask: 0x1,
+                type_mask: MemoryHeapFlags::DEVICE_LOCAL.bits(),
```

Comment:
FYI, technically, a Vulkan implementation has to also support trivial 2D images that are CPU visible. I don't think it's worth pursuing for you though.

Reply:
Mhm, does seem like it could be a bit of a pain to add. Is there a list of minimum requirements or is it scattered over the spec?